src/views/Languages/index.jsx

```diff
@@ -215,69 +187,45 @@ export default class Languages extends Component {
         return this.fetchGithub(searchQuery);
       })
     );
-
-    this.setState({ githubLoading: false });
   };
 
-  handleBugInfoClick = memoizeWith(
-    id => id,
-    async id => {
-      try {
-        const {
-          data: { comments },
-        } = await this.props.client.query({
-          query: commentsQuery,
-          variables: { id },
-          context: { client: 'bugzilla' },
-        });
+  loadNextPage = async () => {
+    await this.setState({ isNextPageLoading: true });
+    await this.load();
+    const githubData = this.props.github;
+    let issues = [];
+    let hasNextPage = false;
 
-        return comments[0].text;
-      } catch (error) {
-        this.setState({ error });
-      }
+    if (githubData && githubData.search) {
```

Comment:
can we make it simpler & move it to the render method?
```suggestion
    let issues = this.props.github && this.props.github.search && uniqBy( 
```